NAMESPACE

```diff
@@ -37,7 +37,6 @@ importFrom(RColorBrewer,brewer.pal)
 importFrom(ape,cophenetic.phylo)
 importFrom(ape,drop.tip)
 importFrom(ape,is.ultrametric)
-importFrom(data.table,melt)
```

Comment:
maybe worth noting that, IINM, the trivial fix here is to just change this line to `importFrom(reshape2, melt)`.

but as {reshape2} is retired, I agree your approach to use `row()`/`col()` directly makes more sense.